src/System.Management.Automation/minishell/api/RunspaceConfigurationTypeException.cs

```diff
@@ -20,12 +20,7 @@ namespace System.Management.Automation.Runspaces
     ///     2. ISerializable
     /// -->
     [Serializable]
-#if CORECLR
-    internal
-#else
-    public
-#endif
-    class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
+    internal class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
```

Comment:
Same kind of changes should be grouped together, for example, this one and #4447